### PR TITLE
adds in episode page and transcript for ep 017

### PR DESCRIPTION
**Pull request by nellshamrell**
Signed-off-by: Nell Shamrell <nellshamrell@gmail.com>

### CODE REVIEW

File: _episodes/017-twir-341-342.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Referenced resources
+- [This Week in Rust 341](https://this-week-in-rust.org/blog/2020/06/02/this-week-in-rust-341/)
```
Comment (jonhoo) [inline _episodes/017-twir-341-342.md:31]: Would it make sense to split this up by 341 and 342 using sub-headings like what we did in https://github.com/rustacean-station/rustacean-station.org/blob/master/_episodes/011-jake-yoshua-stjepan.md ?

File: _episodes/015-twir-339.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: "This Week in Rust - Issue 339"
 date: 2020-05-19T14:30:00Z
 file: https://audio.rustacean-station.org/file/rustacean-station/twir-2020-05-19.mp3
 duration: "07:34"
-length: "7116560"
+length: "5267717"
```
Comment (jonhoo) [inline _episodes/015-twir-339.md:6]: Huh, weird, I wonder why this wasn't caught before..

File: _episodes/017-twir-341-342.md (hunk 1 above)
Comment (nellshamrell) [reply _episodes/017-twir-341-342.md:31]: Good idea - doing that now :)

Comment (nellshamrell) [reply _episodes/017-twir-341-342.md:34]: Done!

**Comment by nellshamrell** [comment]
I don't have a link directly to issue 342 in the resources section because to publish that we need to publish this and it's a classic chicken and egg situation. I'll update this when the published issue of 342 is available :)

File: _episodes/015-twir-339.md (hunk 2 above)
Comment (nellshamrell) [reply _episodes/015-twir-339.md:6]: Not sure myself, I was quite surprised when that came up.

**Comment by nellshamrell** [comment]
@jonhoo ty for the feedback - edits have been made!

File: _episodes/017-twir-341-342.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+- [Custom Types in Diesel](https://kitsu.me/posts/2020_05_24_custom_types_in_diesel)
+- [Fuzzing Sequoia-PGP](https://blog.hackeriet.no/fuzzing-sequoia/)
+- [Sorting algorithms in Rust](https://dev.to/jlkiri/sorting-algorithms-in-rust-1386)
+- [3D boids swimming in perfect harmony: Implementing the boids flocking algorithm in Rust](https://www.reddit.com/r/rust/comments/gsldbi/3d_boids_swimming_along_in_perfect_harmony/)
+- [Aprende Rust en español](https://dev.to/robertohuertasm/aprende-rust-en-espanol-1pea)
+- [A Rust and WASM tutorial on building Bitcoin infrastructure](https://www.youtube.com/watch?v=qaykNPHJcyw)
+- [Crust of Rust: Iterators](https://www.youtube.com/watch?v=yozQ9C69pNs&feature=emb_logo)
+- [Rust and Tell Berlin - May 2020](https://www.youtube.com/watch?v=rpilJV-eIVw&feature=emb_logo)
+
+#### Issue 342
```
Comment (jonhoo) [inline _episodes/017-twir-341-342.md:49]: Nice! Just for future reference, you can also include timestamps here of the form `[@15:09]`, and they will be converted into clickable links that jump in the player on the website. No big need for that in such short episodes, but something to keep in mind :)

**Pull request merged**